docs/fr/point-of-sale-options.md

```diff
@@ -1,4 +1,4 @@
-# Point of Sale option
+# Option de point de vente (PoS)
```

Comment:
Same as Standalone
You can add ("Point of sale = Point de vente") if you want to explain, but PdV isn't use in french e-commerce, where PoS is a lot.